File: PostNApp-UI/src/app/interfaces/post.ts
import { Byte } from "@angular/compiler/src/util";
import { Comment } from "./comment";

export interface Post {
    id: number,
    userId: number,
    image: Byte,
    created: Date,
    title: string,
    body: string, 
    username: string,
    comments: Comment[],
}

File: PostNApp/PostN.DataAccess/Entities/Post.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace PostN.DataAccess.Entities
{
    public partial class Post
    {
        public Post()
        {
            Comments = new HashSet<Comment>();
        }

        public int Id { get; set; }
        public int UserId { get; set; }
        public byte[] Image { get; set; }
        public DateTime Created { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }

        public virtual User User { get; set; }
        public virtual ICollection<Comment> Comments { get; set; }
    }
}
